pkg/analyze/kube_resource.go

```
@@ -108,15 +109,28 @@ func FindResource(kind string, clusterScoped bool, namespace string, name string
 }
 
 func (a *AnalyzeClusterResource) analyzeResource(analyzer *troubleshootv1beta2.ClusterResource, getFileContents getCollectedFileContents) (*AnalyzeResult, error) {
-
 	selected, err := FindResource(analyzer.Kind, analyzer.ClusterScoped, analyzer.Namespace, analyzer.Name, getFileContents)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to find resource")
+		klog.Errorf("failed to find resource: %v", err)
+		return &AnalyzeResult{
+			Title:   a.Title(),
+			IconKey: "kubernetes_text_analyze",
+			IconURI: "https://troubleshoot.sh/images/analyzer-icons/text-analyze.svg",
+			IsFail:  true,
+			Message: "resource does not exist",
+		}, nil
 	}
 
 	actual, err := iutils.GetAtPath(selected, analyzer.YamlPath)
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to get object at path: %s", analyzer.YamlPath)
+		klog.Errorf("failed to get object at path: %s: %v", analyzer.YamlPath, err)
```

Since this error catches an error with the jsonpath, wouldn't be better to return an error since its a syntax error with the spec?

---

how about now?